authentik/core/templatetags/authentik_core.py

```diff
@@ -0,0 +1,24 @@
+"""authentik core tags"""
+
+from django import template
+from django.templatetags.static import static as static_loader
+from django.utils.safestring import mark_safe
+
+from authentik import __version__
+
+register = template.Library()
+
+
+@register.simple_tag()
+def versioned_script(path: str) -> str:
+    """Wrapper around {% static %} tag that supports setting the version"""
+    returned_lines = [
+        f'<script src="{static_loader(path.replace("%v", __version__))}" type="module"></script>',
+        # Legacy method of loading scripts used as a fallback, without the version in the filename
+        # TODO: Remove after 2024.6 or later
+        (
+            f'<script src="{static_loader(path.replace("-%v", ""))}?'
```

Comment:
This is both a fix/workaround for e2e tests but also for rolling deployments as especially during upgrades